### PR TITLE
Refs [TASK] [36682] Create UI class screen

### PR DESCRIPTION
**Pull request by thanhvinh2k**
## Related Tickets
- [#Task 36682: Create UI class screen](https://edu-redmine.sun-asterisk.vn/issues/36682)
## WHAT
- Create Ui for class screen
## Evidence (Screenshot or Video)
classScreen
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/item_layout_class.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="10dp"
+        android:textColor="@android:color/white"
+        android:textSize="@dimen/sp_20"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/imageViewClass"
+        app:layout_constraintTop_toTopOf="parent" />
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (khoaqh-2470) [inline app/src/main/res/layout/item_layout_class.xml]:
```suggestion
</androidx.constraintlayout.widget.ConstraintLayout>

```

File: app/src/main/res/layout/item_layout_class_selected.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="10dp"
+        android:textColor="@android:color/white"
+        android:textSize="@dimen/sp_20"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/imageViewClass"
+        app:layout_constraintTop_toTopOf="parent" />
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (khoaqh-2470) [inline app/src/main/res/layout/item_layout_class_selected.xml]:
```suggestion
</androidx.constraintlayout.widget.ConstraintLayout>

```

File: app/src/main/res/layout/fragment_class.xml [hunk 3]
```diff
@@ -1,8 +1,42 @@
...
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="#081B27"
+    android:padding="10dp">
 
+    <Spinner
+        android:id="@+id/spinner_class"
+        android:layout_width="150dp"
+        android:layout_height="30dp"
```
Comment (datdt-1935) [inline app/src/main/res/layout/fragment_class.xml]: hard code. Không fix cứng width/height cho view nhé

File: app/src/main/res/layout/fragment_class.xml [hunk 4]
```diff
@@ -1,8 +1,42 @@
...
+        android:id="@+id/guideline2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.5" />
+
+    <Spinner
+        android:id="@+id/spinner_difficulty"
+        android:layout_width="150dp"
+        android:layout_height="30dp"
```
Comment (datdt-1935) [inline app/src/main/res/layout/fragment_class.xml]: tương tự

File: app/src/main/res/layout/fragment_class.xml [hunk 5]
```diff
@@ -1,8 +1,42 @@
...
+        android:layout_marginLeft="25dp"
```
Comment (datdt-1935) [inline app/src/main/res/layout/fragment_class.xml]: dùng `marginStart` nhé

File: app/src/main/res/layout/fragment_class.xml [hunk 6]
```diff
@@ -1,8 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="#081B27"
+    android:padding="10dp">
```
Comment (khoaqh-2470) [inline app/src/main/res/layout/fragment_class.xml]: hard code

File: app/src/main/res/layout/fragment_class.xml [hunk 7]
```diff
@@ -1,8 +1,42 @@
...
 
+    <Spinner
+        android:id="@+id/spinner_class"
+        android:layout_width="150dp"
+        android:layout_height="30dp"
+        android:layout_marginRight="25dp"
```
Comment (khoaqh-2470) [inline app/src/main/res/layout/fragment_class.xml]: above

File: app/src/main/res/layout/fragment_class.xml [hunk 8]
```diff
@@ -1,8 +1,46 @@
...
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="@color/firefly"
+    android:padding="@dimen/dp_10">
 
+    <Spinner
+        android:id="@+id/spinner_class"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_class.xml]: convention

File: app/src/main/res/layout/fragment_class.xml [hunk 9]
```diff
@@ -1,8 +1,46 @@
...
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginEnd="@dimen/dp_25"
+        app:layout_constraintDimensionRatio="5:1"
+        app:layout_constraintEnd_toEndOf="@id/guideline2"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintWidth_percent="0.4" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline2"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_class.xml]: convention

File: app/src/main/res/layout/fragment_class.xml [hunk 10]
```diff
@@ -1,8 +1,46 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.5" />
+
+    <Spinner
+        android:id="@+id/spinner_difficulty"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_class.xml]: ****

**Review by datdt-1935 — APPROVED** [review]

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**